### PR TITLE
ebnf show tables

### LINKED ISSUE #17067: sql-statements/sql-statement-show-tables.md

### PR DESCRIPTION
**Pull request by dveeden**
### What is changed, added or deleted? (Required)

Closes #17067

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v8.1 (TiDB 8.1 versions)
- [x] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by hfxsd — APPROVED** [review]

**Review by hfxsd — APPROVED** [review]

**Review by hfxsd — APPROVED** [review]

File: sql-statements/sql-statement-show-tables.md [hunk 1]
```diff
@@ -92,3 +85,4 @@ The `SHOW [FULL] TABLES` statement in TiDB is fully compatible with MySQL. If yo
 * [CREATE TABLE](/sql-statements/sql-statement-create-table.md)
 * [DROP TABLE](/sql-statements/sql-statement-drop-table.md)
 * [SHOW CREATE TABLE](/sql-statements/sql-statement-show-create-table.md)
+* [`information_schema.tables`](/information-schema/information-schema-tables.md)
```
Comment (qiancai) [inline sql-statements/sql-statement-show-tables.md:88]:
```suggestion
* [`INFORMATION_SCHEMA.TABLES`](/information-schema/information-schema-tables.md)
```

File: sql-statements/sql-statement-show-tables.md [hunk 2]
```diff
@@ -6,27 +6,20 @@ aliases: ['/docs/dev/sql-statements/sql-statement-show-tables/','/docs/dev/refer
 
 # SHOW [FULL] TABLES
 
-This statement shows a list of tables and views in the currently selected database. The optional keyword `FULL` indicates if a table is of type `BASE TABLE` or `VIEW`.
+This statement shows a list of tables and views in the currently selected database. The optional keyword `FULL` indicates if a table is of type `BASE TABLE`, `SEQUENCE` or `VIEW`.
```
Comment (qiancai) [inline sql-statements/sql-statement-show-tables.md:9]:
```suggestion
This statement shows a list of tables and views in the currently selected database. The optional keyword `FULL` indicates if a table is of type `BASE TABLE`, `SEQUENCE`, or `VIEW`.
```

**Review by qiancai — APPROVED** [review]
Rest LGTM

**Pull request merged**